cmd/gen-user/main.go

```diff
@@ -21,41 +24,54 @@ NKEYs are sensitive and should be treated as secrets.
 %s
 ------END USER NKEY SEED------
 
-*************************************************************`
+*************************************************************
+`
 
 func main() {
-	creds := flag.Bool("creds", false, "Generate Creds file contents instead.")
-	h := flag.Bool("h", false, "Display this help")
+	creds := flag.Bool("creds", false, "Generates Creds file contents instead.")
+	h := flag.Bool("h", false, "Display this help.")
+	jsManager := flag.Bool("js", false, "Enables JetStream Manager option.")
 	flag.Parse()
 
 	if *h {
 		flag.Usage()
 		return
 	}
 
-	scanner := bufio.NewScanner(os.Stdin)
-	for {
+	inStat, err := os.Stdin.Stat()
+	if err != nil || inStat.Size() == 0 {
 		fmt.Print("Enter Seed(Account NKEY): ")
```

Comment:
It would be best to output errors and informational messages to stderr instead of stdout.

Reply:
Fixed, thanks, all errors now go to stderr.